syft/serde/protobuf/serde.py

```diff
@@ -0,0 +1,355 @@
+from collections import OrderedDict
```

Comment:
`_generate_bufferizers_and_unbufferizers`, `_bufferize`, `_unbufferize`, seem very similar to msgpack's serde.
Although, it seems there's no point to extract that to common code at this time.

Reply:
They're intentionally very, very similar. IIRC, there are a few tweaks to how the data structures are created and what they contain, but there's almost certainly shared code that could be factored out.